### PR TITLE
Sreymom merge sorted list

### PR DESCRIPTION
**Pull request by Sreymompech**
- I use add operation to add list
- I use built-in sorted function
- all tests are passed

### CODE REVIEW

File: functions/merge_sorted_lists.py [hunk 1]
```diff
@@ -1,2 +1,4 @@
 def merge_sorted_lists(l1, l2):
-    pass
+    merged = l1 + l2
+
+    return sorted(merged)
```
Comment (nishatsalsabil) [inline functions/merge_sorted_lists.py:4]: Good job!

File: tests/merge_sorted_lists_test.py [hunk 2]
```diff
@@ -1,2 +1,38 @@
...
+def test_for_1_2_3_and_4_5_6():
+    # Arrange
+    l1 = [1, 2, 3]
+    l2 = [4, 5, 6]
+
+    # Act
+    answer = merge_sorted_lists(l1, l2)
+
+    # Assert
+    assert answer == [1, 2, 3, 4, 5, 6]
```
Comment (nishatsalsabil) [inline tests/merge_sorted_lists_test.py:38]: Awesome tests!

**Review by nishatsalsabil — APPROVED** [review]
Great job overall!

**Comment by Sreymompech** [comment]
Thank for your reviewing

**Pull request merged**